room-manager/src/handlers/productHandler.js

```diff
@@ -0,0 +1,45 @@
+const express = require('express')
+const Product = require('../models/product')
+const productCatalog = require('../productServiceStartup')
+const { validationResult } = require('express-validator');
+
+// POST /products
+// adds a product object to the db
+function addProduct (req, res) { 
+    const errors = validationResult(req)
+
+    //assume at this point the input has been validated
+    if( !errors.isEmpty() ){
+        res.status(400).send({errors: errors.toArray()})
+    }
+
+    try {
+        const product = new Product(req.body.productName, req.body.sku)
```

Comment:
consider DRYing up the access here

```suggestion
        const { productName, sku } = req.body
        const product = new Product(productName, sku)
```